src/master-playlist-controller.js

```diff
@@ -226,6 +227,17 @@ export default class MasterPlaylistController extends videojs.EventTarget {
         this.mainSegmentLoader_.load();
       }
 
+      this.audioPlaylistLoaders_ = [];
+      if (master.mediaGroups && master.mediaGroups.AUDIO) {
+        for (let groupKey in master.mediaGroups.AUDIO) {
+          for (let labelKey in master.mediaGroups.AUDIO[groupKey]) {
+            this.audioPlaylistLoaders_.push(new PlaylistLoader(
```

Comment:
I think it's worth considering making this an object so that you can index into them by URI below.